soc/microchip/mec/Kconfig.soc

```diff
@@ -0,0 +1,13 @@
+# Microchip MEC172x, MEC1501 MCU line
+
+# Copyright (c) 2018 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY_MEC
+	bool
+
+config SOC_FAMILY
+	string
+	default "microchip_mec" if SOC_FAMILY_MEC
```

Comment:
Would need updating here and anywhere else, etc.

Reply:
yup, renamed everywhere throughout the code base